pal/src/configure.cmake

```diff
@@ -1,3 +1,5 @@
+if (NOT CC_TARGET_OS_ANDROID)
```

Comment:
On this note- have we cleaned up this file to make sure that it's checking for stuff that ChakraCore actually needs? When I imported the PAL, I didn't bother to clean this up but if our PAL is diverging, can you make a note to clean up this file at some point?